docs/dictionary/property/beepSound.lcdoc

```diff
@@ -25,6 +25,16 @@ set the beepSound to "internal"
 Example:
 set the beepSound to "vibrate"
 
+Example:
+-- on iOS there is no default beep sound, so you have to set one.
+on preopencard
+   set the beepsound to getBeepSound()
+end preopencard
+
+function getBeepSound
+   return specialFolderPath("resources") & slash & "myBeepSound.m4a"
```

Comment:
Rather than use the 'slash' keyword it is better to use the slash in the quotes:
`return specialFolderPath("resources") & "/myBeepSound.m4a"`

The example would be less case specific if and thus better formatted (and easier to read) if there were no function. Simply have the path within the preopencard handler. Perhaps use this:
```
on preopencard
    put specialFolderPath("resources") & "/myBeepSound.m4a" into tBeepPath
    set the beepsound to tBeepPath
 end preopencard
```